### PR TITLE
Sean completed react-class-components

### PR DESCRIPTION
**Pull request by Sean-Hughes-1**
image

### CODE REVIEW

File: react-class-components/src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+
+function CustomButton(props) {
```
Comment (Zhiwutian) [inline react-class-components/src/index.jsx:4]: so this is a function component, you want to define a class component for this exercise

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
You want to use a class component here instead of a function component

File: react-class-components/src/index.jsx (hunk 1 above)
Comment (Sean-Hughes-1) [reply react-class-components/src/index.jsx]: refactored

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**